libnavigation-core/api/current.txt

```diff
@@ -830,6 +832,27 @@ package com.mapbox.navigation.core.telemetry.events {
     method public com.mapbox.navigation.core.telemetry.events.FeedbackMetadata get();
   }
 
+  public final class UserFeedback {
+    method public String getDescription();
+    method public String getFeedbackId();
+    method public String![]? getFeedbackSubType();
+    method public String getFeedbackType();
+    method public com.mapbox.geojson.Point getLocation();
+    method public String? getScreenshot();
+    method public String getSource();
+    property public final String description;
+    property public final String feedbackId;
+    property public final String![]? feedbackSubType;
+    property public final String feedbackType;
+    property public final com.mapbox.geojson.Point location;
+    property public final String? screenshot;
+    property public final String source;
+  }
+
+  public fun interface UserFeedbackObserver {
```

Comment:
```suggestion
  public fun interface UserFeedbackCallback {
```

This interface is used for one-time operation only, right?

Reply:
Maybe yes? Especially because the one-time is public but not the continuous observing, right?

Reply:
let's add 2: observer + callback and hide observer (for copilot only)

Reply:
Everything is internal, so I think we can keep only callback for now.